lib/fs_utils/pipeline.js

```diff
@@ -18,7 +18,7 @@ const prepareError = (type, stringOrError) => {
     stringOrError;
   const error = new Error(string);
   error.code = type;
-  return error;
+  return Promise.reject(error);
```

Comment:
This is a minor improvement but since the result of this function is always wrapped into a rejection, I thought we could as well do it here